docker-compose.yml

```
@@ -7,7 +7,7 @@ services:
       # OCIS
       OCIS_URL: https://host.docker.internal:9200
       OCIS_INSECURE: true
-      OCIS_LOG_LEVEL: error
+
```

the suggestion was only to undo the unnecessary change. Please, add the log level
```suggestion
      OCIS_LOG_LEVEL: error
```

---

This file should not be touched in this PR